dpnp/dpnp_iface_arraycreation.py

```diff
@@ -807,7 +807,17 @@ def full_like(x1,
     return numpy.full_like(x1, fill_value, dtype, order, subok, shape)
 
 
-def geomspace(start, stop, num=50, endpoint=True, dtype=None, axis=0):
+def geomspace(start,
```

Comment:
the below code behaves differently comparing to numpy (numpy raised `ValueError` exception):
```
# dpnp.geomspace(0, 10, 2)
array([ 0., 10.])

# numpy.geomspace(0, 10, 2)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[114], line 1
----> 1 numpy.geomspace(0, 10, 2)

File <__array_function__ internals>:180, in geomspace(*args, **kwargs)

File /localdisk/work/antonvol/soft/miniconda3/envs/dpnp_py39_ext/lib/python3.9/site-packages/numpy/core/function_base.py:392, in geomspace(start, stop, num, endpoint, dtype, axis)
    390 stop = asanyarray(stop)
    391 if _nx.any(start == 0) or _nx.any(stop == 0):
--> 392     raise ValueError('Geometric sequence cannot include zero')
    394 dt = result_type(start, stop, float(num), _nx.zeros((), dtype))
    395 if dtype is None:

ValueError: Geometric sequence cannot include zero
```

the same for stop=0:
```
# dpnp.geomspace(10, 0, 2)
array([10.,  0.])

# numpy.geomspace(10, 0, 2)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[116], line 1
----> 1 numpy.geomspace(10, 0, 2)

File <__array_function__ internals>:180, in geomspace(*args, **kwargs)

File /localdisk/work/antonvol/soft/miniconda3/envs/dpnp_py39_ext/lib/python3.9/site-packages/numpy/core/function_base.py:392, in geomspace(start, stop, num, endpoint, dtype, axis)
    390 stop = asanyarray(stop)
    391 if _nx.any(start == 0) or _nx.any(stop == 0):
--> 392     raise ValueError('Geometric sequence cannot include zero')
    394 dt = result_type(start, stop, float(num), _nx.zeros((), dtype))
    395 if dtype is None:

ValueError: Geometric sequence cannot include zero
```
and when both `stop` and `start` are zeros.